diagrams/tutorials/cmdline/dcat-dart-run.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can actually optimize SVGs, including removing the diagrams metadata. In this case, I've slightly optimized the one not in the diagrams directory.

Do you think it would be worth keeping the difference and perhaps I can add an explanation to the diagrams README that says do not squash/optimize image/vector files found here?